### PR TITLE
Improving instructions to avoid `verify` confusion

### PR DESCRIPTION
**Pull request by Sequoia**
The solution script uses `console.log`, but this was not specified in the instructions, which *do* mention `stdout`.  So I used `process.stdout.write` & wondered why my output wasn't matching.

Oddly, using `process.stdout.write` [only sometimes](https://dl.dropboxusercontent.com/u/20304987/concat-issue.html) fails.  In any event using `console.log` always works (afaict) so I've added that tip to the instructions.  It seems like a potentially very confusing stumbling block for a beginner & therefor detrimental to the overall lesson, esp since this lesson is about streams and not the vagaries of `stdout` methods.

### CODE REVIEW

**Comment by calvinmetcalf** [comment]
maybe reword this into a more general comment about the differences between console.log and process.stdout.write?

**Comment by Sequoia** [comment]
This may not be an issue anymore, framework updates may have made it so
null & '' match and don't fail the exercise. Anyway idk
